### DISCUSSION TITLE
Cant install Lazer

### DISCUSSION THREAD

**Discussion by Blazzycrafter** [ppy/osu#17649] [Q&A]
### Info
now its installed againg... but it does not starting
### Type

Other

### Bug description

i wanted to start the game but it would not start
and after i tried to reinstall the game the installer deleted all files in appdata/localCosulazer and spit out an error

`exit-error-code 4294967295`

### Screenshots or videos

![image](https://i.imgur.com/vHhSoZW.png)

### Version

latest downloadeble at github

### Logs

have no

**Comment by Blazzycrafter** [comment]
@peppy 
am i stupid or is a crashing installer not a bug?

**Comment by Blazzycrafter** [comment]
@peppy if you just postponed it because of missing info....
why not ask?
i assume that the logs are stored in 
%AppData%/osu/logs
are logs from osu!stable but ...
here:
[performance.log](https://github.com/ppy/osu/files/8410779/performance.log)
[runtime.log](https://github.com/ppy/osu/files/8410780/runtime.log)
[updater.log](https://github.com/ppy/osu/files/8410781/updater.log)
[database.log](https://github.com/ppy/osu/files/8410782/database.log)
[network.log](https://github.com/ppy/osu/files/8410783/network.log)

**Comment by bdach** [comment]
this has been moved into a discussion because there is not enough details to go on. the logs attached are from lazer but are also from april 1 which i presume does not help here if the installer has failed.

please go check for errors that may look related in windows event viewer.

**Reply by Blazzycrafter** [reply]
i retry it...
and now its installed againg....
but when i try to start it nothing happends...

and i dont find something in the event viewer...
-_-

**Reply by caesay** [reply]
Hi, I'm the maintainer of the installer osu! uses on Windows. The logs for Squirrel (the installer) are in `%localappdata%\SquirrelClowdTemp\Squirrel.log`, can you have a look and see if there is anything relevant there?

**Comment by Blazzycrafter** [comment]
nevermind
a computer restart fixes the most problems
yeah...
fixed

Caelan Sayler ***@***.***> schrieb am Do., 7. Apr. 2022,
09:45:

**Reply by caesay** [reply]
It would be great if you are willing to check for and share that log file (Squirrel.log) anyway. Ideally I would like to improve that dialog you saw with a much better message describing what went wrong.
